COPYRIGHT.txt

```diff
@@ -485,7 +485,7 @@ License: Apache-2.0
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
  .
-     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Are you sure it's fine to edit the urls in the text of the license?

Reply:
For the reference, they still use HTTP on the same page that this links to. 🙃 And they don't redirect HTTP to HTTPS, for some reason.

https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
I have a feeling that we should have Apache fix it on their end.

Reply:
Their website actually contains a mix of HTTP and HTTPS links, this one is HTTPS:

![apache](https://user-images.githubusercontent.com/1646875/130369228-808871c7-f71d-4c95-bb1b-cfe2bf9ed709.png)